### ISSUE TITLE
Escape sequences in PyCharm's console on Windows

### ISSUE THREAD

**Issue by seguri** [linux-china/jetbrains-just-plugin#28]
Hi,

When I execute one of the commands in my Justfile using your plugin, the escape sequences are displayed literally:

```
just lint
←[1mset_env && black --line-length 120 .←[0m
←[1;31merror←[0m: ←[1mRecipe `lint` failed on line 7 with exit code 127←[0m
```

Is there a way to customize how your plugin invokes just (so that I can try adding something like `--no-color`) or do you know how to fix this in PyCharm? I've already set up UTF-8 in Settings > Editor > General > Console > Default encoding.

Thanks

**Comment by linux-china** [comment]
It's fixed on version 0.5.2, and `JUST_COLOR=never` added as env variable.

**Comment by linux-china** [comment]
Fixed with version 0.5.2.
